site/docs/community.md

```diff
@@ -49,11 +49,8 @@ Issues are tracked in GitHub:
 
 ## Slack
 
-We use the [#iceberg](https://the-asf.slack.com/archives/CF01LKV9S) channel on
-the ASF's Slack workspace. You get can an invitation to join this workspace at
-<https://s.apache.org/slack-invite>. Please note that this link occasionally
-breaks when Slack does an upgrade. If you encounter problems using it, please
-let us know by sending an email to <dev@iceberg.apache.org>.
+You can join the community via the [apache-iceberg](https://join.slack.com/t/apache-iceberg/) Slack workspace.
```

Comment:
the https://join.slack.com/t/apache-iceberg/ link currently doesn't work for me.

It redirects me to https://apache-iceberg.slack.com/join and shows

> There’s been a glitch…
>
> We’re not quite sure what went wrong. You can go back, or try looking on our Help Center if you need a hand.

Reply:
does https://apache-iceberg.slack.com/ work for you?

Reply:
it does -- shows the log in page, as i do  not have an account

i also cannot create a new account, since i don't have an `@apache.org` email address.

signup page's footprint says

> Don’t have an @apache.org email address?
Contact the workspace administrator at apache-iceberg for an invitation.

Reply:
I don't have an `@apache.org` email as well but I could "Sign in with Google"

Reply:
"Sign in with Google" > choose my account >  "{my-address-email}@gmail.com doesn’t have an account on this workspace."

@nastra did you have an account before?

Reply:
sorry my bad, of cource "Sign in with Google" would only work when you have an account (and I already had created one). You can just create one via `Create an account`in the top right corner

Reply:
@nastra 

`Create an account` gives a page where i have options:

- "Continue with Google" -- i didn't try that before, so thanks for making me try that now. After authenticating with Google, i am back on Slack and Slack shows error message "The email address must match one of the domains listed below. Please try another email."
- type e-mail for signup, but this requires `@apache.org` address


![image](https://user-images.githubusercontent.com/144328/127616377-a132af0c-67af-4c63-acd2-adc600d40fb2.png)